src/modules/bragg/functions.cpp

```diff
@@ -24,28 +23,22 @@ bool BraggModule::calculateBraggTerms(ProcessPool &procPool, Configuration *cfg,
                                       const double qMax, Vec3<int> multiplicity, bool &alreadyUpToDate)
 {
     // Check to see if the arrays are up-to-date
-    auto braggDataVersion = GenericListHelper<int>::retrieve(cfg->moduleData(), "BraggVersion", "", -1);
+    auto braggDataVersion = cfg->moduleData().retrieve<int>("BraggVersion", "", -1);
     alreadyUpToDate = braggDataVersion == cfg->contentsVersion();
     if (alreadyUpToDate)
         return true;
 
     // Realise the arrays from the Configuration
-    auto &braggKVectors = GenericListHelper<Array<KVector>>::realise(cfg->moduleData(), "BraggKVectors");
-    auto &braggReflections = GenericListHelper<Array<BraggReflection>>::realise(cfg->moduleData(), "BraggReflections", "",
-                                                                                GenericItem::InRestartFileFlag);
-    Array2D<double> &braggAtomVectorXCos =
-        GenericListHelper<Array2D<double>>::realise(cfg->moduleData(), "BraggAtomVectorXCos");
-    Array2D<double> &braggAtomVectorYCos =
-        GenericListHelper<Array2D<double>>::realise(cfg->moduleData(), "BraggAtomVectorYCos");
-    Array2D<double> &braggAtomVectorZCos =
-        GenericListHelper<Array2D<double>>::realise(cfg->moduleData(), "BraggAtomVectorZCos");
-    Array2D<double> &braggAtomVectorXSin =
-        GenericListHelper<Array2D<double>>::realise(cfg->moduleData(), "BraggAtomVectorXSin");
-    Array2D<double> &braggAtomVectorYSin =
-        GenericListHelper<Array2D<double>>::realise(cfg->moduleData(), "BraggAtomVectorYSin");
-    Array2D<double> &braggAtomVectorZSin =
-        GenericListHelper<Array2D<double>>::realise(cfg->moduleData(), "BraggAtomVectorZSin");
-    auto &braggMaximumHKL = GenericListHelper<Vec3<int>>::realise(cfg->moduleData(), "BraggMaximumHKL");
+    auto &braggKVectors = cfg->moduleData().realise<Array<KVector>>("BraggKVectors");
+    auto &braggReflections =
+        cfg->moduleData().realise<Array<BraggReflection>>("BraggReflections", "", GenericItem::InRestartFileFlag);
+    Array2D<double> &braggAtomVectorXCos = cfg->moduleData().realise<Array2D<double>>("BraggAtomVectorXCos");
+    Array2D<double> &braggAtomVectorYCos = cfg->moduleData().realise<Array2D<double>>("BraggAtomVectorYCos");
```

Comment:
In keeping with other examples in the files, I guess these could also be auto declared 
```suggestion
    auto &braggAtomVectorYCos = cfg->moduleData().realise<Array2D<double>>("BraggAtomVectorYCos");
```